airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/initialsync/MssqlInitialReadUtil.java

```diff
@@ -167,7 +163,7 @@ public static List<AutoCloseableIterator<AirbyteMessage>> getCdcReadIterators(fi
         true,
         firstRecordWaitTime,
         subsequentRecordWaitTime,
-        AirbyteDebeziumHandler.QUEUE_CAPACITY,
+        queueSize,
```

Comment:
I'll just use this opportunity to ask questions:

From my understanding, this underlying queue is a BlockingQueue only used outside of debezium - in other words we do not change debezium properties like `max.queue.size` right? 

And since this basically defines the size of the queue, it could mean - 
1. the larger the number is, we could probably benefit from having fewer bottleneck issues
2. the lower the number is, the less preserved memory it could cost.

Is that right? In that case, I feel we could benefit from gathering data on the max queue size used in the sync